recipes/csbschema/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "csbschema" %}
+{% set version = "1.1.2" %}
+{% set python_min = "3.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/csbschema-{{ version }}.tar.gz
+  sha256: 551bec314c98c30f0d76f4ee705196dff6d5304e634d01f97e72140a02ed38ee
+
+build:
+  entry_points:
+    - csbschema = csbschema.command.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >={{ python_min }}
+    - jsonschema >=4.21.0,<4.22.dev0
```

Comment:
Can you use `jsonschema-with-format` since upstream asks for it:
https://github.com/CCOMJHC/csbschema/blob/main/pyproject.toml#L21

and there is an feedstock output that covers the dependencies:
https://github.com/conda-forge/jsonschema-feedstock/blob/main/recipe/meta.yaml#L51C11-L71

Reply:
Oops! Grayskull doesn't' seem to know how to handle this subtly. I should've caught that. Done.